test/workers/scanner_test.go

```diff
@@ -119,8 +132,8 @@ func TestScannerDryRun(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	// Sleep for two cycles.
-	time.Sleep(2 * workers.SleepBetweenScans())
+	// Sleep for a while, giving a chance to the scanner to pick the skylink up.
+	time.Sleep(cyclesToWait * workers.SleepBetweenScans())
```

Comment:
same, we should be using build reties instead of bare sleeps if there are timing inconsistencies.

Reply:
Addressed in https://github.com/SkynetLabs/pinner/pull/34#discussion_r886854260